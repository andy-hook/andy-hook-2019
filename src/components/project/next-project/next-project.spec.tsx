import React from "react"
import { mockProjectsData } from "@mock-data"

import NextProject from "./next-project"
import { shallowWithTheme } from "@test-utils"

describe("<NextProject />", () => {
  it("renders correctly", () => {
    const tree = shallowWithTheme(
      "dark",
      <NextProject nextProjectItem={mockProjectsData.brandwatch} />
    )
    expect(tree).toMatchSnapshot()
  })
})
