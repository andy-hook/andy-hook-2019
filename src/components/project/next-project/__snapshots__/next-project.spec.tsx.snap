// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NextProject /> renders correctly 1`] = `
<div
  className="sc-bxivhb FWKFj"
>
  <div
    className="sc-bwzfXH hDOArp"
  >
    Brandwatch
  </div>
</div>
`;
